test/auction_test.js

```diff
@@ -2,9 +2,11 @@ const Freeport = artifacts.require("./Freeport.sol");
 const SimpleAuction = artifacts.require("SimpleAuction");
 const TestERC20 = artifacts.require("TestERC20");
 const log = console.log;
-const {deployProxy} = require('@openzeppelin/truffle-upgrades');
-const {expectEvent, expectRevert, constants, time} = require('@openzeppelin/test-helpers');
+const { deployProxy } = require('@openzeppelin/truffle-upgrades');
```

Comment:
No need to include style changes in this PR. The code was written with Intellij defaults. You may add a prettier config or such that aligns with the existing style.